common/lib/xmodule/xmodule/js/src/video/06_video_progress_slider.js

```diff
@@ -181,7 +164,8 @@ function () {
 
     function onSlide(event, ui) {
         var time = ui.value,
-            duration = this.videoPlayer.duration();
+            duration = (this.config.endTime !== null) ? this.config.endTime : this.videoPlayer.duration();
+        duration = Math.min(duration, this.videoPlayer.duration());
```

Comment:
why do you need this Math.min? Is it possible to set a shorter duration than your end time minus your start time?